packages/app-desktop/gui/MenuBar.tsx

```diff
@@ -555,9 +516,6 @@ function useMenu(props: Props) {
 						selector: 'performClose:',
 					}, {
 						type: 'separator',
-					}, {
-						label: _('Templates'),
-						submenu: templateItems,
```

Comment:
Did you test the app after having made your changes? Because if you remove this, there will be two successive two separators in the menu, so you need to remove one separator. Please test the code. Check all the parts where you've changed something.

Reply:
Actually, I did test it and I couldn't see two separators in the UI. But anyways, removed it.
![Screenshot_20210625_002635](https://user-images.githubusercontent.com/36989112/123328535-52081880-d559-11eb-9fee-6fa3f7971647.png)